auth-api/src/auth_api/services/affiliation.py

```diff
@@ -457,6 +457,10 @@ def _combine_affiliation_details(details):
             if 'nrNumber' in business and (nr_num := business['nrNumber']):
                 if business['nrNumber'] in name_requests:
                     business['nameRequest'] = name_requests[nr_num]['nameRequest']
+                    # Update the draft type if the draft NR request is for amalgamation
+                    if business.get('draftType', None) \
+                            and business['nameRequest']['request_action_cd'] == NRActionCodes.AMALGAMATE.value:
+                        business['draftType'] = CorpType.ATMP.value
```

Comment:
I expected to see this code alongside other code that sets TMP or RTMP ...?

Reply:
It is here line 450. `sets TMP or RTMP` but at the moment, we don't have the NameRequests.
So, I updated the draftType when combine the NR

Reply:
I see. I still think that AAs are fundamentally the same as IAs and Regs (even including NRs) but if you and Travis think this is OK then I'm OK with this, too.